examples/compiled/area_overlay.vg.json

```diff
@@ -11,28 +11,8 @@
       "name": "source_0",
       "url": "data/stocks.csv",
       "format": {"type": "csv"},
-      "transform": [{"type": "filter", "expr": "datum.symbol==='GOOG'"}]
-    },
-    {
-      "name": "data_0",
-      "source": "source_0",
-      "transform": [
-        {"type": "formula", "expr": "toDate(datum[\"date\"])", "as": "date"},
-        {"type": "formula", "expr": "toNumber(datum[\"price\"])", "as": "price"}
-      ]
-    },
-    {
-      "name": "data_1",
-      "source": "source_0",
-      "transform": [
-        {"type": "formula", "expr": "toDate(datum[\"date\"])", "as": "date"},
-        {"type": "formula", "expr": "toNumber(datum[\"price\"])", "as": "price"}
-      ]
-    },
-    {
-      "name": "data_2",
-      "source": "source_0",
       "transform": [
+        {"type": "filter", "expr": "datum.symbol==='GOOG'"},
```

Comment:
@domoritz  Why is filter before parsing here?

Reply:
Well, the symbol does not need to be parsed to it's fine.